<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Тренировки</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>67</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Клиенты</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Тип тренировки</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxWorkoutType">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>270</y>
     <width>251</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>67</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Дата</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>460</y>
     <width>67</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Время</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>400</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>460</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>67</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Тренер</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxTrainers">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>340</y>
     <width>251</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addWorkoutButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>560</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>271</width>
     <height>131</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::NoSelection</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
